.github/workflows/ci-header-test.yml

```
@@ -0,0 +1,14 @@
+name: SMARTS CI Header Test
+
+on: [push]
+
+env:
+  venv_dir: .venv
+jobs:
+  test-header:
+    runs-on: ubuntu-18.04
+    container: huaweinoah/smarts:v0.4.3-pre
+    steps:
+      - name: test-header
+        run: |
+          make header-test
```

The error is: No rule to make target `header-test`.
I think it was unable to find the make. Hence, cd to the appropriate directory first.
```suggestion
          cd $GITHUB_WORKSPACE
          make header-test
```